docs/UserGuide.md

```diff
@@ -27,24 +27,16 @@ For more experienced users who know what they are looking for and want to quickl
 
 Alternatively, you may use the Windows shortcut **Ctrl + F** or the Mac shortcut **Command + F** to find specific keywords.
 
-### Glossary of definitions
-
-**_Command-line Interface (CLI)_**: A user interface that allows users to interact with a system through text commands.
-
-**_Graphical User Interface (GUI)_**: A user interface that allows users to interact with a system through graphical icons.
-
-**_Parameter_**: Input to a command's field.
-
-**_JavaScript Object Notation (JSON)_**: A text-based representation of structured data that is used to save contacts as files in *Teletubbies*.
-
 ### Glossary of icons and symbols
```

Comment:
Should we keep the Glossary at the start so that users are familiar with the definitions before going through the guide? Since we're advocating for top-to-bottom navigation